test/shopify.test.js

```diff
@@ -303,6 +304,55 @@ describe('Shopify', () => {
         });
     });
 
+    // Test base mixin
+    it('injects the api version to the path if provided (1/4)', () => {
```

Comment:
Can you move these in tests of the respective resources? For example this should be in `shop.test.js`.
We should ideally do it for all resources, but I think I'm fine doing it only for some just to cover all cases.

Reply:
Alright. I also removed the checks after the requests complete since it's really just testing if the proper URL gets hit. The tests were just copy/pasted from existing tests anyways. Is this fine?

Reply:
I would keep it. I understand it is redudand but it does not harm and makes the test work like a real example.

One final request: can you also please make the newly added `shopify.graphql()` support versions? Thank you.